test.sh

```diff
@@ -1,5 +1,10 @@
+# set -e
+
 go test .
 
+export PATH=`pwd`:$PATH
 export LANG=C
+
 rm -f examples/*.err
-./cram --quiet examples/env.t
+
+cram examples/*.t tests/*.t
```

Comment:
Why cram? Shouldn't this be grill?

Reply:
Yeah, I it should. I originally named it cram to keep the test suite as unchanged as possible. There's also a clash between output grill binary and grill/ directory. Should I move exec sources to standard cmd/grill/ and move grill/ contents back to root?

Reply:
Done